util/cron/test-gpu-ex-cuda-12.interop.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# GPU native testing on a Cray EX (using none for CHPL_COMM)
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common.bash
+source $CWD/common-hpe-cray-ex.bash
+
+
+# We need 12.4 for the stream test because the CUDA driver on pinoak
+# only supports PTX for 12.4, untill the driver is updated, we need to
+# stick with 12.4 instead of 12.5
+module load cuda/12.4  # default is CUDA 12
```

Comment:
The `# default is CUDA 12` is not adding much anymore (it was probably added back when we had to fall back to 11 in the past), and I recommend dropping.

Reply:
Meant for it to say `default is CUDA 12.5`